src/app/shared/form/builder/form-builder.service.ts

```diff
@@ -161,6 +165,8 @@ export class FormBuilderService extends DynamicFormService {
     } else {
       this.typeBindModels[id] = new BehaviorSubject<DynamicFormControlModel>(model);
     }
+    // Notify that the typeBindModels configuration has changed.
+    this.emitTypeBindChange();
```

Comment:
seems a little bit overkill to create a method for just one line, you could insert `his.typeBindFieldsChange$.next();` directly here